activities/tests/test_host.py

```diff
@@ -11,9 +12,12 @@ class GrantRemoveHostTest(django.test.TestCase):
 
     def setUp(self):
         """Set up the common URL and create an activity."""
+        DAYSDELTA = 7
```

Comment:
You should have `_` to separate the words

Reply:
Fixed